protos/sources/vega/data/v1/spec.proto

```diff
@@ -19,6 +19,10 @@ message PropertyKey {
   string name = 1;
   // type is the type of the property.
   Type type = 2;
+  // An optional decimal place to be be applied on the provided value
+  // valid only for PropertyType of Type DECIMAL and INTEGER
```

Comment:
```suggestion
  // An optional decimal place to be applied on the provided value
  // valid only for PropertyType of type DECIMAL and INTEGER
```

Reply:
Just wondering out of curiosity - why is there a spec.proto and a data.proto with the same types?

Reply:
Both corrections are added, thanks!
The types are created for different purpose - one is used in the core and the other one in the datanode part.
Usually for the datanode part are used types that correspond with existing ones in the core - but simplified.
Because the core ones need to handle the core respective logic, and the ones in the datanode - do not.